### ISSUE TITLE
Multiple Beacons of Light in ICC not working

### ISSUE THREAD

**Issue by bogeldi** [dalaranwow/dalaran-wow#4937]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
If you have 2 Holy Paladins and you each put a Beacon of Light on different people inside ICC they sometimes stop working at all. 
They either don't work at all or only the last casted Beacon of Light works. You can also stack Beacon on one person and then they work, which should also not happen.

**Expected behaviour**: Tell us what should happen instead
Normally each Beacon should work as if they are not casted on the same person. 

**Steps to reproduce the problem**:

1.  Get 2 Holy Paladins in ICC
2.  Cast Beacon on different persons, and try to heal through them

**Include proofs for this behaviour**
[Beacon of Light](http://www.wowhead.com/spell=53563/beacon-of-light)
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Duplicate with #4743
